docs/src/pages/reference/QueryClient.md

```diff
@@ -131,7 +131,7 @@ await queryClient.prefetchQuery(queryKey)
 
 **Options**
 
-The options for `prefetchQuery` are exactly the same as those of [`useQuery`](#usequery).
+The options for `prefetchQuery` are exactly the same as those of [`useQuery`](#usequery) except the following: `enabled, refetchInterval, refetchIntervalInBackground, refetchOnWindowFocus, refetchOnReconnect, notifyOnChangeProps, notifyOnChangePropsExclusions, onSuccess, onError, onSettled, useErrorBoundary, select, suspense, keepPreviousData, placeholderData`; which are stictly for useQuery and useInfiniteQuery. You can check the [source code](https://github.com/tannerlinsley/react-query/blob/361935a12cec6f36d0bd6ba12e84136c405047c5/src/core/types.ts#L83) for more clarity.
```

Comment:
what about `fetchQuery?` there, it still says:

> The options for `fetchQuery` are exactly the same as those of [`useQuery`](#usequery).

which is also not true. My suggestion was to add this exceptions to `fetchQuery` and then have all others reference that:

```diff
- The options for `prefetchQuery` are exactly the same as those of [`useQuery`](#usequery).
+ The options for `prefetchQuery` are exactly the same as those of [`fetchQuery`](#queryclientfetchquery).

Reply:
Yeah, I like this idea, too 👍